Classes/common/touchdb/TDRemoteRequest.m

```diff
@@ -267,6 +253,18 @@ - (void)connectionDidFinishLoading:(NSURLConnection *)connection
     [self respondWithResult:self error:nil];
 }
 
+- (void)handleData:(NSData *)data {
+    [self receivedData:data];
```

Comment:
Fixed in e199317 by renaming the methods in the protocol and making the associated changes.